integrations/spark-extensions-basetests/src/main/java/org/projectnessie/spark/extensions/AbstractNessieSparkSqlExtensionTest.java

```diff
@@ -733,7 +733,8 @@ void testCompaction(String branchName, String tableName)
             tableName);
 
     // re-written files count is 4 and the added files count is 2
-    soft.assertThat(compactionResult.get(0)).startsWith(4, 2);
+    soft.assertThat(compactionResult.get(0))
+        .satisfiesAnyOf(l -> assertThat(l).startsWith(4, 2), l -> assertThat(l).startsWith(5, 2));
```

Comment:
Why is it not certain? Could you update the comment on line 735?

Reply:
It changes on Iceberg main